src/com/asjaHas/qamp/oop/homework5/task1/CurrentAccount.java

```diff
@@ -0,0 +1,30 @@
+package com.asjaHas.qamp.oop.homework5.task1;
+
+public class CurrentAccount extends Account {
+
+    public CurrentAccount(final Owner owner, final String serialNumber, final int passcode, final double balance) {
+        super(owner, serialNumber, passcode, balance);
+    }
+
+    @Override
+    public double withdraw(final double amount) {
+        setBalance(getBalance() - amount);
```

Comment:
This is fine, but you should have made the setBalance() method `protected` so that someone can't alter an account directly

you could have had a method like `changeBalance(double value)` in the Account class,  and provide positive or negative value, based on whether you withdraw or not to alter the balance to avoid exposing a setter